router/voteCallbacks.go

```diff
@@ -73,6 +74,37 @@ func (r *Router) submitEnvelope(request routerRequest) {
 	}
 }
 
+func (r *Router) getStats(request routerRequest) {
+	stats := new(models.VochainStats)
+
+	stats.BlockHeight = uint32(r.vocapp.State.Header(true).Height)
+	stats.BlockTimeStamp = int32(r.vocapp.State.Header(true).Timestamp)
+
+	stats.EntityCount = r.Scrutinizer.EntityCount()
+
+	votes, err := r.Scrutinizer.GetEnvelopeHeight([]byte{})
+	if err != nil {
+		log.Warnf("could not count vote envelopes: %s", err)
+	}
+	stats.EnvelopeCount = votes
+
+	stats.ProcessCount = r.Scrutinizer.ProcessCount([]byte{})
```

Comment:
We should cache the process count the same way we cache the envelope count, else it might become very costly.

Reply:
I think you can get this from `r.vocinfo`, that would be an easy solution. Up to you.